src/identity.js

```diff
@@ -782,10 +769,12 @@ export class Identity extends EventEmitter {
      */
     logoutUrl(redirectUri = this.redirectUri) {
         assert(isUrl(redirectUri), `logoutUrl(): redirectUri is invalid`);
-        return this._spid.makeUrl('logout', {
-            response_type: 'code',
-            redirect_uri: redirectUri
-        });
+        const params = { redirect_uri: redirectUri };
+        if (this._sessionService) {
```

Comment:
This seems to be breaking change, how about adding new option to constructor https://github.com/schibsted/account-sdk-browser/blob/master/src/identity.js#L90 ? `singleSiteLogin` with default set to `false` maybe?